example-mlem-get-started/code/README.md

```diff
@@ -0,0 +1,39 @@
+# MLEM Get Started
+
+This is an auto-generated repository for use in MLEM
+[Get Started](https://mlem.ai/doc/get-started). It is a step-by-step quick
+introduction into basic MLEM concepts.
+
+The main branch contains only the code needed for the tutorial. Results of each step can be found in tags, and resulting state in correctponding branches.
+
+
+🐛 Please report any issues found in this project here -
+[example-repos-dev](https://github.com/iterative/example-repos-dev).
+
+## Installation
+
+Python 3.7+ is required to run code from this repo.
+
+To be able to commit and push your results please fork and then clone this repo
+
+```console
+$ gh repo fork https://github.com/iterative/example-mlem-get-started
+$ cd example-mlem-get-started
+```
+
+Now let's install the requirements. But before we do that, we **strongly**
+recommend creating a virtual environment with a tool such as
+[virtualenv](https://virtualenv.pypa.io/en/stable/):
+
+```console
+$ python3 -m venv .env
+$ source .env/bin/activate
+$ pip install -r requirements.txt
+```
+
+## Existing stages
+
+This project with the help of the Git tags reflects the sequence of actions that
```

Comment:
Git tags - let's make it a link to https://github.com/iterative/example-mlem-get-started/tags 

Also, it looks it's a mess now - I see a few tags that start with the same number

Reply:
Sorry, which ones do start with the same number?
<img width="1266" alt="image" src="https://user-images.githubusercontent.com/6797716/177951484-1fb452c5-e26f-4789-8f1b-b0563b2582d4.png">

Reply:
yes, please disregard that comment about the number

Reply:
> would be nice to explain that there are two branches I think

Done :)

Reply:
https://github.com/iterative/mlem.ai/pull/132#issuecomment-1185452401